Makefile

```diff
@@ -0,0 +1,37 @@
+TEST?=$$(go list ./... | grep -v 'vendor')
```

Comment:
this might need tweaking as it as a copy and paste from the `terraform-provider-aws` repo but wanted thoughts on adding it first before making the changes

Reply:
I'm using the goreleaser plugin for now so this is unnecessary and can be removed